### ISSUE TITLE
Bad query can kill the influxdb process

### ISSUE THREAD

**Issue by matthewmcneely** [influxdb/influxdb#3768]
Executing this query crashed the server.

```select value, value from units```

Environment:
* influxdb 0.9.2
* 3.16.0-30-generic #40~14.04.1-Ubuntu SMP

```
panic: interface conversion: interface is float64, not map[string]interface {}

goroutine 97 [running]:
github.com/influxdb/influxdb/tsdb.(*limitedRowWriter).processValues(0xc208108460, 0xc208248000, 0x507, 0x2710, 0x0)
	/root/go/src/github.com/influxdb/influxdb/tsdb/engine.go:758 +0xc63
github.com/influxdb/influxdb/tsdb.(*limitedRowWriter).Flush(0xc208108460)
	/root/go/src/github.com/influxdb/influxdb/tsdb/engine.go:693 +0xba
github.com/influxdb/influxdb/tsdb.(*RawExecutor).execute(0xc208198150, 0xc2080a7c20)
	/root/go/src/github.com/influxdb/influxdb/tsdb/engine.go:149 +0x2d7
created by github.com/influxdb/influxdb/tsdb.(*RawExecutor).Execute
	/root/go/src/github.com/influxdb/influxdb/tsdb/engine.go:93 +0x5a
```

**Comment by otoolep** [comment]
Thanks @matthewmcneely -- I'll take a look.

Can I see some sample data you are inserting?

**Comment by matthewmcneely** [comment]
Sure, it's pretty a pretty straightforward table.

```select * from units```

```
time	value
2015-08-04T20:03:35.005157982Z	50
2015-08-04T20:04:35.093645554Z	49.15
2015-08-04T20:05:35.161594409Z	50
2015-08-04T20:06:35.229053511Z	48.3
2015-08-04T20:07:35.311157954Z	50
2015-08-04T20:08:35.378330131Z	49.15
2015-08-04T20:09:35.447028519Z	47.5
2015-08-04T20:10:35.525772789Z	48.3
2015-08-04T20:11:35.59519855Z	50
2015-08-04T20:12:35.66284025Z	50
2015-08-04T20:13:35.74535156Z	48.3
```

**Comment by matthewmcneely** [comment]
The error seems to be with the dup'd query term, using a non-existent tag or value yields the expected error.

**Comment by otoolep** [comment]
Can you try with a 0.9.3-rc1? I cannot reproduce with master code, and we made many fixes to the query engine since 0.9.2.

https://influxdb.com/download/index.html

**Comment by matthewmcneely** [comment]
Sure, I was planning on trying that version out in a few days on another machine. I'll update this issue afterwards.

**Comment by beckettsean** [comment]
Verified fixed in a recent 0.9.3 nightly:

```
> select value, value from thing
name: thing
-----------
time			value	value
1970-01-01T00:15:00Z	50	50
1970-01-01T00:15:10Z	60	60
```
